docs/docker_instructions.md

```diff
@@ -115,7 +108,7 @@ There are readings to help with the exercises. Refer here for any questions you
 
 ### Exercise 1
 #### Part A Readings
-Whalesay is a program that given some text, will print out an ASCII whale that is saying the text. It is based on a program called cowsay.
+Cowsay is a program that given some text, will print out an ASCII cow that is saying the text. It is based on a program called cowsay.
```

Comment:
There's several lingering "whale" references in this file, `grep -rIi whale` will find them